plugins/kotlin/idea/tests/testData/formatter/trailingComma/whenEntry/WhenEntry.after.kt

```diff
@@ -13,16 +13,14 @@ fun foo(x: Any) {
 
     when (x) {
         Comparable::class, Iterable::class,
-        String::class, /*// trailing comma*/
-        -> println(1)
+        String::class /*// trailing comma*/ -> println(1)
 
         else -> println(3)
     }
 
     when (x) {
         Comparable::class, Iterable::class,
-        String::class, /*// trailing comma*/
-        -> println(1)
+        String::class /*// trailing comma*/ -> println(1)
```

Comment:
The same as above, the trailing comma is missing

Reply:
Droped this change